### PR TITLE
Review timeouts API documentation

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
This is the doc review of https://github.com/ballerina-platform/lsp4intellij/pull/33.

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,31 +28,29 @@ protocol.
 - [**Useful Links**](#useful-links)
 
 
-## How To Use
+## How to use LSP4IntelliJ
```
Comment (NipunaRanasinghe) [inline README.md]:
```suggestion
## How To Use
```

File: README.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
-# [Lsp4IntelliJ](#sp4intellij) - Language Server Protocol Support for Jetbrains Plugins
+# [LSP4IntelliJ](#sp4intellij) - Language Server Protocol Support for the  Jetbrains Plugins
 
 [![Build Status](https://travis-ci.com/ballerina-platform/lsp4intellij.svg?branch=master)](https://travis-ci.com/NipunaRanasinghe/lsp4intellij)
 [![](https://jitpack.io/v/ballerina-platform/lsp4intellij.svg)](https://jitpack.io/#ballerina-platform/lsp4intellij)
 [![GitHub last commit](https://img.shields.io/github/last-commit/ballerina-platform/lsp4intellij.svg)](https://github.com/ballerina-platform/lsp4intellij/commits/master)
 
-**Lsp4IntelliJ** provides language server support for IntelliJ IDEA and other Jetbrains IDEs.
+**Lsp4IntelliJ** provides is a client library, which provides language server support for IntelliJ IDEA and other Jetbrains IDEs.
```
Comment (nixel2007) [inline README.md]: Provides provides

File: README.md (hunk 1 above)
Comment (NipunaRanasinghe) [reply README.md]: Shall we keep this as it is? :) Since it is a common title used by another plugins and libraries as well

File: README.md (hunk 2 above)
Comment (praneesha) [reply README.md]: Fixed.

File: README.md (hunk 1 above)
Comment (praneesha) [reply README.md]: +1

**Pull request merged**